SCClassLibrary/Common/Core/LanguageConfig.sc

```diff
@@ -43,4 +43,12 @@ LanguageConfig {
 		_LanguageConfig_setPostInlineWarnings
 		^this.primitiveFailed
 	}
+	*defaultPathsExcluded {
```

Comment:
Wouldn't it be more consistent with the rest of the code, to call these `excludeDefaultPaths`? Everywhere else, e.g. postInlineComments, functions like these have the same name as in YAML

Reply:
And, it would be nice to have these two functions documented LanguageConfig's help file as well